geom/encoding/geojson/geojson.go

```diff
@@ -117,3 +129,18 @@ type FeatureCollection struct {
 	Type     featureCollectionType `json:"type"`
 	Features []Feature             `json:"features"`
 }
+
+func closePolygon(p geom.Polygon) {
+	for i := range p {
+		if len(p[i]) < 3 {
+			log.Warn("encounted polygon with less than 3 points")
+			continue
+		}
```

Comment:
I'm not sure we should even check for this. I think we should just add the last point. You should check to see if it's as at least one point, otherwise, we have a panic.

Reply:
I was going to initially check for 1 point but decided to check for 3. I'm fine with checking for a single point.